helper/error-handler.ts

```diff
@@ -56,30 +50,21 @@ export function errorHandler(caughtError: Error | HttpError | AxiosError | Runti
   /**
    * The error message looks like: [404] Not Found. User does not exist
    */
-  switch (error.statusCode) {
-    case 400:
-      throw `[${error.statusCode}] ${error.name ? `${error.name}. ` : ''}${error.message || 'Bad request'}`;
-    case 401:
-      throw `[${error.statusCode}] ${error.name ? `${error.name}. ` : ''}${error.message || 'Unauthorized'}`;
-    case 403:
-      throw `[${error.statusCode}] ${error.name ? `${error.name}. ` : ''}${error.message || 'Forbidden'}`;
-    case 404:
-      throw `[${error.statusCode}] ${error.name ? `${error.name}. ` : ''}${error.message || 'Not Found'}`;
-    case 409:
-      throw `[400] ${error.name ? `${error.name}. ` : ''}${error.message || 'Conflict'}`;
-    case 422:
-      throw `[${error.statusCode}] ${error.name ? `${error.name}. ` : ''}${error.message || 'Unprocessable Entity'}`;
-    case 429:
-      throw `[400] ${error.name ? `${error.name}. ` : ''}${error.message || 'Too Many Requests'}`;
-    case 500:
-      throw `[${error.statusCode}] ${error.name ? `${error.name}. ` : ''}${error.message || 'Internal Server Error'}`;
-    case 502:
-      throw `[${error.statusCode}] ${error.name ? `${error.name}. ` : ''}${error.message || 'Bad Gateway'}`;
-    case 503:
-      throw `[500] ${error.name ? `${error.name}. ` : ''}${error.message || 'Service Unavailable'}`;

```

Comment:
Serverless doesn't support 409, 429, and 503 error so we replace them with 400 and 500 error. But now you just use statusCode. We need a function that will return the proper status code for unsupported errors.